webcompat/issues.py

```diff
@@ -23,12 +23,17 @@
 PRIVATE_REPO_URI = app.config['PRIVATE_REPO_URI']
 
 
-def unmoderated_issue():
+def moderation_template():
     """Gets the placeholder data to send for unmoderated issues."""
-    # TODO: Replace this with something meaningful.
-    # See https://github.com/webcompat/webcompat.com/issues/3137
-    summary = 'Placeholder in-moderation title.'
-    body = 'Placeholder in-moderation body.'
+
+    summary = 'In the moderation queue.'
+    body = """<p>This issue has been put in the moderation queue. A human
+will review if the message meets our current
+<a href="https://www.mozilla.org/en-US/about/legal/acceptable-use/">
+acceptable use</a> guidelines.
+It will probably take a couple of days depending on the backlog.
+Once it has been reviewed, the content will be either made public
+or deleted.</p>"""
```

Comment:
```suggestion
or deleted.</p>'''
```

Reply:
Ahh so I wasn't sure if you're referring to this part as well @karlcow. 
Didn't find any occurrences of `'''some text'''` being used with other than docstring. 

There is `"""some text"""` is used if `form.py` for filling out issue details:

https://github.com/webcompat/webcompat.com/blob/master/webcompat/form.py#L338
https://github.com/webcompat/webcompat.com/blob/master/webcompat/form.py#L526

Reply:
indeed we seem to have inconsistencies. One day we will probably need to bit the bullet and just fix it with black. Maybe worth adding as topic to Berlin.